tests/installation/ipxe_install.pm

```diff
@@ -169,6 +170,12 @@ sub run {
         select_console 'installation';
         save_screenshot;
 
+        # We have textmode installation via ssh and the default vnc installation so far
+        if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
+            type_string_slow('DISPLAY= ') if check_var('VIDEOMODE', 'text');
+            enter_cmd_slow("yast.ssh");
+        }
+
```

Comment:
From my perspective this can be done like this:
```
enter_cmd_slow('DISPLAY=yast.ssh') if check_var('VIDEOMODE', 'text');
enter_cmd_slow('yast.ssh') if check_var('VIDEOMODE', 'ssh-x');
```

You do not need to judge whether VIDEOMODE is text twice.

Reply:
Yes, agree